.storybook/components/DetailPageHeaderStory.js

```diff
@@ -51,9 +53,14 @@ storiesOf('DetailPageHeader', module)
           {...overflowMenuItemProps}
           itemText="Delete App"
           isDelete
-          isLastItem
```

Comment:
This isn't related to this pull request, but I noticed that this prop no longer exists in the `OverflowMenuItem`, so I removed it.